frontend/src/sass/pages/_contact.scss

```diff
@@ -1,3 +0,0 @@
-/* =================================
-   PAGES / CONTACT 
-   ================================= */
```

Comment:
ah, now I realise is deleted, so no empty file.. sorry.. forget my other comments :)